src/Oscoin/Data/OscoinTx.hs

```diff
@@ -283,6 +284,17 @@ insertAccount acc@Account{..} = WorldState.insert (accountKey accountId) (Accoun
 deleteAccount :: CBOR.Serialise (PublicKey c) => AccountId c -> WorldState c -> WorldState c
 deleteAccount id = WorldState.delete (accountKey id)
 
+-- | Transfer balance between two accounts. Uses 'creditAccount' and 'debitAccount'.
+transferBalance
+    :: (CBOR.Serialise (PublicKey c))
+    => AccountId c
+    -> AccountId c
```

Comment:
Apart from using a `newtype`, which would make working with these heavier, can you think of any other way we can prevent these two `AccountId` from being swapped? I can totally foresee how that can happen, and when our code will grow complex we will probably soon forget "which one was which" if we don't recall the type signature.

Reply:
Hmm - can't think of any way, though the current order encodes the directionality of the transfer, ie. `From -> To`. I expect anyone using this function to triple-check :smile:

Reply:
Added some doc to the params, should help.